acceptance/lib/puppet/acceptance/mount_utils.rb

```diff
@@ -0,0 +1,56 @@
+module Puppet
+  module Acceptance
+    module MountUtils
+
+      # Return the absolute path to the filesystem table file.
+      # @param host [String] hostname
+      # @return [String] path to the filesystem table file.
+      def filesystem_file(host)
+        if host['platform'] =~ /aix/
+          '/etc/filesystems'
+        else
+          '/etc/fstab'
+        end
+      end
+
+      # Return a standard filesystem type to use when creating filesysytems.
+      # @param host [String] hostname
+      # @return [String] filesystem type.
+      def filesystem_type(host)
+        if host['platform'] =~ /aix/
+          'jfs2'
+        else
+          'ext3'
```

Comment:
This should also be more guarded now that it is in a helper. See comment for line 12.